test/e2e/daemon_restart.go

```diff
@@ -183,8 +183,7 @@ func getContainerRestarts(c *client.Client, ns string, labelSelector labels.Sele
 	return failedContainers, containerRestartNodes.List()
 }
 
-// Flaky issues #17829, #19023
-var _ = Describe("DaemonRestart [Disruptive] [Flaky]", func() {
+var _ = Describe("DaemonRestart [Disruptive]", func() {
```

Comment:
ah, we should totally be tolerant to reastarts in soak-serial but yeah, makes sense